tokio/src/io/driver/mod.rs

```diff
@@ -303,100 +333,3 @@ impl Direction {
         }
     }
 }
-
-#[cfg(all(test, loom))]
-mod tests {
```

Comment:
@hawkw I couldn't think of a way to update these tests that makes sense. Resources don't access the `ScheduleIo` via token anymore. Do you have thoughts on this?

Reply:
IIRC, these tests were largely intended to test that the generational indexing worked correctly. Since the slab itself doesn't handle the generation, the new slab tests don't really cover this case...it would still be nice to be able to test token uniqueness.

I'd have to think a bit more about _how_ we'd do that, though.

Reply:
There are other tests ensuring there are no "false positives" with notifying I/O resources.